packages/cli/src/__tests__/commands/DebugInfo.test.ts

```diff
@@ -149,14 +149,106 @@ describe('debug', () => {
       false
 
       -- CI detected? --
-      No
+      false
+
+    `)
+  })
+
+  it('should succeed when env vars are set to empty', async () => {
+    ctx.fixture('example-project/prisma')
+
+    // Make sure all env vars are set to ''
+    const envVarsSetToEmptyString = Object.fromEntries(Object.keys(envVars).map((key) => [key, '']))
+    Object.assign(process.env, envVarsSetToEmptyString)
+    // To make sure the terminal is always detected
+    // as non interactive, locally and in CI
+    process.env.TERM = 'dumb'
+
+    const result = await DebugInfo.new().parse([])
+
+    expect(cleanSnapshot(result as string)).toMatchInlineSnapshot(`
+      -- Prisma schema --
+      Path:  REDACTED_PATH
```

Comment:
2 spaces intentional?

Reply:
It's fixed, it was just the test snapshots anyway